nwb_conversion_tools/utils/specdatachunkiterator.py

```diff
@@ -0,0 +1,74 @@
+"""Authors: Cody Baker, Oliver Ruebel."""
+from typing import Iterable
+import numpy as np
+
+from hdmf.data_utils import AbstractDataChunkIterator, DataChunk
+
+
+class SpecDataChunkIterator(AbstractDataChunkIterator):
+    """DataChunkIterator that lets the user specify chunk shapes."""
+
+    def __init__(self, data: Iterable, full_shape: tuple, chunk_shape: tuple, dtype: str):
+        self.data = data
+        self.full_shape = full_shape
+        self.chunk_shape = chunk_shape
+        self.__curr_timestep = 0
+        self.__curr_channel_idx = 0
+        self.__dtype = DataChunk(data=np.empty(shape=(0, 0), dtype=dtype))
```

Comment:
``self.__dtype`` does not appear to be used. Also should this not be ``self.__dtype=np.dtype``?

Reply:
Fixed now